napari/layers/points/points.py

```diff
@@ -1833,70 +1548,797 @@ def _set_highlight(self, force=False):
+        border color of the marker border for the next point to be added or the currently
+        selected point.
+    current_face_color : str
+        Face color of the marker border for the next point to be added or the currently
+        selected point.
+    out_of_slice_display : bool
+        If True, renders points not just in central plane but also slightly out of slice
+        according to specified point marker size.
+    selected_data : Selection
+        Integer indices of any selected points.
+    mode : str
+        Interactive mode. The normal, default mode is PAN_ZOOM, which
+        allows for normal interactivity with the canvas.
+
+        In ADD mode clicks of the cursor add points at the clicked location.
+
+        In SELECT mode the cursor can select points by clicking on them or
+        by dragging a box around them. Once selected points can be moved,
+        have their properties edited, or be deleted.
+    face_color_mode : str
+        Face color setting mode.
+
+        DIRECT (default mode) allows each point to be set arbitrarily
+
+        CYCLE allows the color to be set via a color cycle over an attribute
+
+        COLORMAP allows color to be set via a color map over an attribute
+    border_color_mode : str
+        border color setting mode.
+
+        DIRECT (default mode) allows each point to be set arbitrarily
+
+        CYCLE allows the color to be set via a color cycle over an attribute
+
+        COLORMAP allows color to be set via a color map over an attribute
+    shading : Shading
+        Shading mode.
+    antialiasing: float
+        Amount of antialiasing in canvas pixels.
+    canvas_size_limits : tuple of float
+        Lower and upper limits for the size of points in canvas pixels.
+    shown : 1-D array of bool
+        Whether each point is shown.
+
+    Notes
+    -----
+    _view_data : array (M, D)
+        coordinates of points in the currently viewed slice.
+    _view_size : array (M, )
+        Size of the point markers in the currently viewed slice.
+    _view_symbol : array (M, )
+        Symbols of the point markers in the currently viewed slice.
+    _view_border_width : array (M, )
+        border width of the point markers in the currently viewed slice.
+    _indices_view : array (M, )
+        Integer indices of the points in the currently viewed slice and are shown.
+    _selected_view :
+        Integer indices of selected points in the currently viewed slice within
+        the `_view_data` array.
+    _selected_box : array (4, 2) or None
+        Four corners of any box either around currently selected points or
+        being created during a drag action. Starting in the top left and
+        going clockwise.
+    _drag_start : list or None
+        Coordinates of first cursor click during a drag action. Gets reset to
+        None after dragging is done.
+    """
+
+    @rename_argument(
+        "edge_width", "border_width", since_version="0.5.0", version="0.6.0"
+    )
```

Comment:
I start thinking that rename argument my got list of names... 

but maybe some update properties work may help (to reduce number of changes).

This is loud thinking, not strict request for for changes.